package/windows/hyperkube.ps1

```diff
@@ -943,9 +943,39 @@ function init {
         try {
             $splited = $hostname -split "\."
             if ($splited.Length -lt 1) {
-                $NodeName = $splited[0]
+                $nn = $splited[0]
             }
         } catch {}
+
+        ## repair Get-GcePdName method ##
+        $getGcePodNameCommand = Get-Command -Name Get-GcePdName -ErrorAction Ignore
+        if (-not $getGcePodNameCommand) {
+            print "Can't find Get-GcePodName, try to repair ..."
+
+            $dllDownloadURL = "https://github.com/pjh/gce-tools/raw/master/GceTools/GetGcePdName/GetGcePdName.dll"
```

Comment:
How will it work in airgapped installs?

Reply:
Unless the fix is for GCE specifically, and GCE implies non-airgapped installs. Then we should make sure this code is never called for non-gce cloud providers and on prem setups

Reply:
@alena1108 , the `Get-GcePodName` is only used to GCE. The Windows host of GCE [hasn't supported](https://github.com/kubernetes/kubernetes/issues/74674) `Get-GcePodName` yet, but the next Windows Kuberentes Cluster Service will [support this function](https://github.com/kubernetes/kubernetes/blob/f49fe2a7504c40684573f3e4928e8b0bfdfb2b30/cluster/gce/windows/k8s-node-setup.psm1#L782-L795). So I only deal with it when using the GCE cloud provider: 

1. Use `Get-Command -Name Get-GcePdName -ErrorAction Ignore` to detect `Get-GcePdName` existing or not
1. If not, download `GetGcePdName.dll` and import it

Should we need to consider AirGap on GCE? cc @loganhz